lms/static/sass/multicourse/_sysadmin.scss

```diff
@@ -0,0 +1,47 @@
+.mgmt-commands-body #select-command{
+    border: solid 4px black;
+    padding: 20px;
+
+}
+
+.mgmt-commands-body .full-width{
+    width: 100%;
+}
+
+.mgmt-commands-body .bordered-fieldset{
+    border: solid 4px black;
```

Comment:
Color names can actually vary implementation-to-implementation.

Can we use hex or rgb(a) values here and throughout?